File: 0283-move-zeroes/solution.cpp
class Solution {
public:
    void moveZeroes(vector<int>& nums) {
        int indx=0;
        int cnt = 0;
        while (cnt <nums.size() ){
            if (nums[indx]==0){
                nums.insert(nums.end(),0);
                nums.erase(nums.begin()+indx);
                cnt++;
            }else{
                cnt++;
                indx++;
            }
        }
        

        }
        
};

File: 0283-move-zeroes/solution.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        f = 0
        s = 0
        while f < len(nums):
            if nums[f] !=0:
                temp = nums[s]
                nums[s] = nums[f]
                nums[f] = temp
                s+=1
            f+=1